source/leuven/services/openondemand.rst

```diff
@@ -0,0 +1,250 @@
+the folder using the Folder Explorer (click 'View Files' on top or 'Open Dir' at the bottom). As explained above, you can edit or remove any file, create new files,
+upload new files... These files will be present in each job you create from this template, which means you fully adapt this to the standard set-up you use for your
+jobs.
+
+Jobs
+~~~~
+
+The functioning of creating jobs is a bit similar to how you create new templates. Whatever method you choose, you will create a new folder for each job, this time
+located at ``$VSC_DATA/ondemand/data/projects/default/``. The job folders will be numbered according to the order you have created them in. **Do not change this folder name as long as you plan on using it from the job menus, as this will break the linking here.** When removing a job, the folder will be deleted as well. 
+
+To create a job, press the 'New Job' button and choose the option that suits your needs the most. You will get a new item in your job list for each job you've created. 
+Again, you can edit, remove and add files like you want to create a custom job by going to the File Explorer (click 'Edit Files' or 'Open Dir') or by directly clicking 
+the file names. The 'Open Editor' button in the 'Submit Script' overview also allows you to edit the job script directly.
+
+Using the 'Job Options' button, you can add some more specifications to your job:
+
+- Name: this will specify a name in the job composer list. This will not be your job name. The actual job name is the one that will be specified in the job script. If you do not specify a name there, you will see that that job gets the name 'sbatch' in the 'Active Jobs' menu.
+- Cluster: there is no need to change this, as you only can specify 'wice'.
+- Specify job script: if you have multiple job scripts in the directory, you can specify which one to run
+- Account: here you can specify which account to use. **Be aware that this will overwrite the account you might have specified in your job script**
+
+Everything should now be set up to start a job. Any job can be started by clicking 'Submit'. You can stop it at any time by clicking 'Stop'. You cannot use the 
+'Submit' job to start the exact same job multiple times. You can use the 'New Job - From Selected Job' option for this. If you delete any of the jobs, you also remove
+the folder that it is associated with. 
+
+Clusters
+--------
+
+When selecting 'Clusters - Login Server Shell Access' you will get a terminal window in a new browser tab. You will arrive on one of the Genius login nodes, which
+you can use as you are used to, including the option to submit jobs to wICE. As with the Genius login nodes, this means that this shell is not meant for any 
+calculations. If you would like to perform calculations in an interactive shell, you should be using the :ref:`interactive shell app<interactive_shell>`
```

Comment:
```suggestion
calculations. If you would like to perform calculations in an interactive job, you should be using the :ref:`interactive shell app<interactive_shell>`
```


Also: I don't seem to have such an interactive shell app. I only see JupyterLab, RStudio, TensorBoard, VSCode, and the two that are work in progress (ParaView, CryoSparc).

Reply:
OK, I see below that that app is still WIP as well.